Delectio/translator-frontend
# Setup react boilerplate

#### What does this PR do?

#### Description of Task to be completed?
- Setup babel
- Setup webpack
- Add demo react component

#### How should this be manually tested?
- Clone the repository
- On your terminal, start the server by running `npm run start`
- The server should start on open the page on the browser
- You should see a webpage titled "App Home"

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?(if applicable)
None

#### Screenshots
<img width="501" alt="Screenshot 2019-09-12 at 2 44 27 PM" src="https://user-images.githubusercontent.com/42358974/64789201-da2b5280-d56b-11e9-8e15-d3173fcb271b.png">
